action.yaml

```
@@ -82,8 +82,8 @@ inputs:
   artifact-type:
     description: 'input artifact type (image, fs, repo, archive) for SBOM generation'
     required: false
-  github-pat:
-    description: 'GitHub Personal Access Token (PAT) for submitting SBOM to GitHub Dependency Snapshot API'
+  github-token:
+    description: 'Authentication token to enable sending SBOM scan results to GitHub Dependency Graph. Can be either a GitHub Personal Access Token (PAT) or GITHUB_TOKEN'
     required: false
```

```suggestion
    required: false
    default: ${{ github.token }}
```
We could add this to make the configuration even simpler, not sure if that is a good idea, but it how it's done for https://github.com/actions/checkout/blob/main/action.yml for example.

---

@simar7 should this be part of the PR?